Kickstarter-iOS/Views/Controllers/SetYourPasswordViewController.swift

```diff
@@ -0,0 +1,260 @@
+  }
+
+  // MARK: - Styles
+
+  public override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.contextLabel
+      |> contextLabelStyle
+
+    _ = self.rootStackView
+      |> baseStackViewStyle
+      |> loginRootStackViewStyle
+
+    _ = self.newPasswordLabel
+      |> textFieldLabelStyle
+
+    _ = self.confirmPasswordLabel
+      |> textFieldLabelStyle
+
+    _ = self.newPasswordTextField
+      |> textFieldStyle
+      |> \.accessibilityLabel .~ self.newPasswordLabel.text
+      |> \.attributedPlaceholder %~ { _ in settingsAttributedPlaceholder("") }
+
+    _ = self.confirmPasswordTextField
+      |> textFieldStyle
+      |> \.accessibilityLabel .~ self.confirmPasswordLabel.text
+      |> \.attributedPlaceholder %~ { _ in settingsAttributedPlaceholder("") }
+
+    _ = self.saveButton
+      |> savePasswordButtonStyle
+      |> \.isEnabled .~ false
+  }
+
+  // MARK: - Bind View Model
+
+  public override func bindViewModel() {
+    self.viewModel.outputs.contextLabelText
+      .observeForUI()
+      .observeValues { [weak self] text in
+        self?.contextLabel.text = text
+      }
+
+    self.viewModel.outputs.newPasswordLabel
+      .observeForUI()
+      .observeValues { [weak self] text in
+        self?.newPasswordLabel.text = text
+      }
+
+    self.viewModel.outputs.confirmPasswordLabel
+      .observeForUI()
+      .observeValues { [weak self] text in
+        self?.confirmPasswordLabel.text = text
+      }
+
+    self.viewModel.outputs.saveButtonIsEnabled
+      .observeForUI()
+      .observeValues { [weak self] isEnabled in
+        self?.saveButton.isEnabled = isEnabled
+      }
+  }
+
+  // MARK: - Functions
+
+  private func configureViews() {
+    _ = self.view
+      |> \.autoresizingMask .~ .flexibleHeight
+      |> \.backgroundColor .~ .ksr_white
+      |> \.clipsToBounds .~ true
+
+    _ = (self.scrollView, self.view)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = (self.rootStackView, self.scrollView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    _ = ([
+      self.contextLabel,
+      self.newPasswordLabel,
+      self.newPasswordTextField,
+      self.confirmPasswordLabel,
+      self.confirmPasswordTextField,
+      self.saveButton
+    ], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    self.rootStackView.setCustomSpacing(Styles.grid(7), after: self.contextLabel)
+    self.rootStackView.setCustomSpacing(Styles.grid(3), after: self.confirmPasswordTextField)
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.rootStackView.widthAnchor.constraint(equalTo: self.view.widthAnchor),
+      self.newPasswordTextField.heightAnchor.constraint(greaterThanOrEqualToConstant: 44),
+      self.confirmPasswordTextField.heightAnchor.constraint(greaterThanOrEqualToConstant: 44),
+      self.saveButton.heightAnchor.constraint(greaterThanOrEqualToConstant: 48)
```

Comment:
Maybe use this style of constants (found inside `CommentsViewController` and others) when defining static heights:
```
private enum Layout {
  enum Composer {
    static let originalHeight: CGFloat = 80.0
  }

  enum ErrorState {
    static let cellHeightMuliplier: CGFloat = 0.70
  }
}
```